sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +558,151 @@ public async Task ValidateMergeCertificate()
             CollectionAssert.AreEqual(mergedServerCertificate.Cer, completedServerCertificate.Cer);
         }
 
-        // Backup Restore
-        // GetCertificates
-        // GetCertificatsIncludePending
-        // GetCertificateVersions
-        // GetDeletedCertificates
-        // GetUpdatePolicy
-        // IssuerCrud
-        // ContactsCrud
+        [Test]
+        public async Task VerifyGetIssuer()
+        {
+            string issuerName = WellKnownIssuerNames.Self;
+
+            string providerName = "CN=Azure SDK";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+
+            Response<CertificateIssuer> getIssuerResponse = await Client.GetIssuerAsync(WellKnownIssuerNames.Self);
```

Comment:
Tip: this works, but there's an implicit cast operator for `Response<T>` -> `T`, so we typically document something like `CertificateIssuer getIssuer = await Client.GetIssuerAsync(...);`